src/Illuminate/Foundation/Testing/CrawlerTrait.php

```diff
@@ -93,6 +93,26 @@ public function post($uri, array $data = [], array $headers = [])
     }
 
     /**
+     * Visit the given URI with a POST request with content type of application/json.
+     *
+     * @param  string  $uri
+     * @param  array  $data
+     * @param  array  $headers
+     * @return $this
+     */
+    public function postJson($uri, array $data = [], array $headers = [])
+    {
+        $headers['CONTENT_TYPE'] = 'application/json';
+        $headers['Accept'] = 'application/json';
```

Comment:
I was thinking this too. Wouldn't it be better to be a bit more general and let the user still specify accept headers, etc for these cases?